src/lib/engine/calculator.js

```diff
@@ -0,0 +1,3 @@
+export function fullUrl({id}) {
+  return `https://www.getguesstimate.com/calculators/${id}`
```

Comment:
I feel like the absolute base url should be somewhere else, this seems quite messy to me.